docs/pics/audformat-misc-table.dot

```diff
@@ -3,12 +3,12 @@ digraph structs {
     rankdir=LR
     node[shape=record]
 
-    header [label="Header | ... | <table> table | <split> split | media | <rater> rater | <scheme> scheme | ... "]
+    header [label="Header | ... | <misctable> misc table | <table> table | <split> split | media | <rater> rater | <scheme> scheme | ... "]
```

Comment:
There is no arrow from `media` to `misc table` but actually we support setting a `media ID`: https://audeering.github.io/audformat/api.html#audformat.MiscTable

Reply:
I thought it makes only sense to have media ID for recordings, but for a misc table we don't have recordings.

Reply:
Yes, but currently we can set a media ID, so I think this should be reflected also by the diagram.

Reply:
I added the media link for misc tables and updated the screenshot in the description of this pull request.